frontend/src/Components/Account/Login.js

```diff
@@ -18,11 +20,13 @@ export default function Login() {
   };
   const handleSubmit = (e) => {
     e.preventDefault();
+    setLoading(true);
     axios
       .post("http://localhost:8000/user/login", { ...form })
       .then((resp) => {
         localStorage.setItem("access_token", resp.data.data.access_token);
         navigate("/home");
+        setLoading(false);
         window.location.reload();
       })
       .catch((err) => {
```

Comment:
A bug:
Catching the error should also setLoading(false).

Currently if you login with wrong credentials, it will show `Please wait...` infinitely.

```suggestion
      .catch((err) => {
      setFormError(true);
      setLoading(false);
      ....
      
```